addons/miscMedical/config.cpp

```diff
@@ -12,6 +12,28 @@ class CfgPatches {
     };
 };
 
+class ace_medical_treatment {
+    class Medication {
+        class Naloxone {
+            painReduce = 0;
+            hrIncreaseLow[] = {0, 0};
+            hrIncreaseNormal[] = {0, 0};
+            hrIncreaseHigh[] = {0, 0};
+            timeInSystem = 180;
+            timeTillMaxEffect = 1;
+            maxDose = 0.1;
+            onOverDose = QFUNC(Naloxone);
+            incompatibleMedication[] = {};
+            viscosityChange = 0;
+        };
+        class Painkillers;
+        class Pkblister: Painkillers {};
+    };
+};
```

Comment:
If we're spinning off all the configs into their own file, then `ace_medical_treatment` should live in a file called `ACE_Medical_Treatment.hpp`

Reply:
Sounds good - wasn't sure where I should've put it, but this makes sense. Done!